Sources/Aardvark/BugReporting/LogStoreAttachmentGenerator.swift

```diff
@@ -124,10 +129,19 @@ public final class LogStoreAttachmentGenerator: NSObject {
 
         let fileName = logsFileName(for: logStoreName, fileType: "txt")
 
+        let recentErrorSummary = logMessages
+            .lazy
```

Comment:
Um, I'll be lazy: is `.lazy` doing anything here, given the `isEmpty` call just below? (I don't know!)